### PR TITLE
New header for docs.

### PR DESCRIPTION
**Pull request by veegandhi**
## Description

- New header format
- Introduction, logos, bells and whistles

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Screenshots (if appropriate):
Screen Shot 2020-02-13 at 4 06 19 pm

Screen Shot 2020-02-13 at 4 06 27 pm

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -1 +1,24 @@
-* text eol=lf
+* text=auto
```
Comment (veegandhi) [inline .gitattributes]: Thanks @lteacher for sorting these out 🎉

File: www/components/Header/index.jsx [hunk 2]
```diff
@@ -1,30 +1,37 @@
...
+          <Contributors />
+          <Popover popoverContent="Github" placement="bottom" theme="dark">
+            <a href="https://github.com/Adslot/adslot-ui">
+              <img src="./assets/github.png" />
+            </a>
+          </Popover>
+        </div>
+      </PageTitle>
+    </div>
+  </React.Fragment>
```
Comment (lightbringer1991) [inline www/components/Header/index.jsx]: do we need this `React.Fragment`?

File: .gitattributes (hunk 1 above)
Comment (lteacher) [reply .gitattributes]: Yeah lol err i did just randomly copy from somewhere else so dunno if it misses anything but prob just has extra stuff that we dont need which am not too worried about anyways.

File: .gitattributes [hunk 3]
```diff
@@ -1 +1,24 @@
...
+*.html       text eol=lf
+*.md         text eol=lf
+*.properties text eol=lf
+*.yml        text eol=lf
+*.json       text eol=lf
+*.config     text eol=lf
+*.inc        text eol=lf
+*.manifest   text eol=lf
+*.rdf        text eol=lf
+*.jade       text eol=lf
```
Comment (adityasabnis) [inline .gitattributes]: `*.pug` instead of `*.jade` ?

Comment (adityasabnis) [reply .gitattributes]: just checked that there are neither pug nor jade files in adslot-ui.
Don't need any changes

**Comment by adityasabnis** [comment]
Merging 🚀

**Pull request merged**